### PR TITLE
Refactor plasma functions

### PR DESCRIPTION
**Pull request by thec00n**
Addresses #5 
Also added sendTypedTx as cli option

### CODE REVIEW

File: src/omgcli.ts [hunk 1]
```diff
@@ -358,6 +358,69 @@ export class OMGCLI {
...
+
+      let tx: any = {
+        transactionType: 1,
+        inputs,
+        outputs,
+        metadata:
+          "0x0000000000000000000000000000000000000000000000000000000000001337"
+      };
+      const newTx = await this.addFeesToTx(tx);
+      if (newTx) {
```
Comment (kalouo) [inline src/omgcli.ts:413]: Non-blocking `try/catch` could be suited here. 

``` typescript
let newTx
try {
  newTx = await ... 
} catch {
  console.log( .... ) 
}

**Review by kalouo — APPROVED** [review]
LGTM

File: src/omgcli.ts (hunk 1 above)
Comment (thec00n) [reply src/omgcli.ts:413]: Why is it better to have a none blocking try/catch block? Should this be handled at a higher level?

Comment (kalouo) [reply src/omgcli.ts:413]: Sorry. I missed the word `but` in there. 😆  (Non-blocking comment, but ... )

I had in mind the scenario of failure to fetch fees. Could be better handled in `addFeesToTx` perhaps. It just seemed to me that the `if` statement could be replaced by a `try/catch` somewhere along the chain - but nothing too essential here!

**Pull request merged**